tests/framework/matcher/phase.go

```diff
@@ -68,31 +68,39 @@ func (p phaseMatcher) FailureMessage(actual interface{}) (message string) {
 	if helper.IsNil(actual) {
 		return fmt.Sprintf("object does not exist")
 	}
+	u, err := helper.ToUnstructured(actual)
+	if err != nil {
+		return err.Error()
+	}
 	expectedPhase := getExpectedPhase(p.expectedPhase)
 	if helper.IsSlice(actual) {
-		return fmt.Sprintf("expected phases to be in %v but got %v", expectedPhase, collectPhasesForPrinting(actual))
+		return fmt.Sprintf("%s %s expected phases to be in %v but got %v", u.GetKind(), u.GetName(), expectedPhase, collectPhasesForPrinting(actual))
```

Comment:
I would use `%s/%s` to match what kubectl uses for example.

Reply:
Thanks, will fix, and add tests